.github/workflows/release.yml

```diff
@@ -38,11 +30,21 @@ jobs:
       - name: Create CHECKSUMS
         if: steps.branch.outputs.BRANCH_NAME == 'main'
         run: ( cd dist ; sha256sum -b carrier* > SHA256SUM.txt )
+      - name: Generate Changelog
+        uses: heinrichreimer/github-changelog-generator-action@v2.1.1
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          pullRequests: "false"
+          onlyLastTag: "true"
+          stripGeneratorNotice: "true"
+          issuesWoLabels: "true"
+          stripHeaders: "true"
       - name: Release Carrier
         uses: softprops/action-gh-release@v1
         if: steps.branch.outputs.BRANCH_NAME == 'main'
         with:
           files: ./dist/*
-          body: ${{steps.changelog.outputs.changelog}}
+          body_path: ./CHANGELOG.md
+          prerelease: "true"
```

Comment:
Why a prerelease?  Should we manually check an convert to a final release? I'm ok with that, I'm just making sure I got this right.

Reply:
Since the pipeline does automatic releases and all our other releases were pre-releases I thought we should stick with it until we decided differently. So yes, I think we should manually switch it when we think we are ready for beta.